at_backupkey_flutter/lib/services/backupkey_service.dart

```
@@ -9,17 +9,15 @@ class BackUpKeyService {
     return _singleton;
   }
 
-  late AtClientService atClientService;
-
-  Future<String?> _getAESKey(String atsign) async {
-    return await atClientService.getAESKey(atsign);
+  static Future<String?> _getAESKey(String atsign) async {
+    return await KeyChainUtil.getAESKey(atsign);
   }
 
-  Future<Map<String, String>> getEncryptedKeys(String atsign) async {
+  static Future<Map<String, String>> getEncryptedKeys(String atsign) async {
     var result;
     try {
-      result = await atClientService.getEncryptedKeys(atsign);
-      result[atsign] = await this._getAESKey(atsign);
+      result = await KeyChainUtil.getEncryptedKeys(atsign);
+      result[atsign] = await _getAESKey(atsign);
     } catch (e) {
       result = {};
```

Let's add a log here. We are swallowing an exception.